.pipelines/stages/jobs/capi-packaging-job.yml

```diff
@@ -55,10 +55,7 @@ jobs:
   - name: os
     value: ${{ parameters.os }}
   - name: feed_name
-    ${{ if and(eq(parameters.cuda_version, '12.2'), eq(parameters.ep, 'cuda')) }}:
```

Comment:
The main `ORT-Nightly` feed now have CUDA12 packages produced. I'm not sure if we should keep using `ort-cuda-12-nightly`

Reply:
We can change to ort-nightly feed and use ort-nightly-gpu.
Should we keep the option for cuda 11? We will need to change to ort-cuda-11-nightly feed for that.

Reply:
If we want to always test the latest nightly, we should primarily use `ORT-Nightly` which has the latest packages. From the names, `ort-cuda-11-nightly` and `ort-cuda-12-nightly` should also have latest package. But right now the packages in `ort-cuda-12-nightly` are 3-months old. I'm quite confused by which feed is the "correct" one to use.